django-backend/fecfiler/transactions/schedule_f/utils.py

```diff
@@ -10,6 +10,7 @@ def add_schedule_f_contact_fields(instance: Transaction, representation=None):
         add_candidate_contact(data, instance.contact_2, "payee", False)
     if instance.contact_3:
         data["payee_committee_id_number"] = instance.contact_3.committee_id
+        data["payee_committee_name"] = instance.contact_3.committee_id
```

Comment:
I feel like this might be a typo?

Reply:
Follow-up: loading a saved Schedule F transaction shows that the committee id is being saved in the committee name field
![image](https://github.com/user-attachments/assets/808b86a9-a845-4b16-9677-080407055744)

Reply:
great catch! @Elaine-Krauss-TCG  #eagle-eye.  Thanks for testing things.  Pushed this fix up.